starter-code/pizza.js

```diff
@@ -1 +1,73 @@
 // Write your Pizza Builder JavaScript in this file.
+$( document ).ready(function(){
+
+
+  // $(".pep").hide();
+  // $(".mushroom").hide();
+  // $(".green-pepper").hide();
+  // $(".sauce").removeClass("sauce-white");
+  // $(".crust").removeClass("crust-gluten-free");
+  // $(".price li").hide();
+
+  $(".btn-pepperonni").data("price", 1);
+  $(".btn-mushrooms").data("price", 1);
+  $(".btn-green-pepers").data("price", 1);
+  $(".btn-sauce").data("price", 3);
+  $(".btn-crust").data("price", 5);
+
+  function calculateTotalPrice(){
+    var allTheButtons = $('.btn');
+    var totalPrice= 10;
+
+    for(var i = 0; i < $('.btn').length; i++){
+      if(allTheButtons[i].hasClass('active')){
+        price += allTheButtons.eq(i).data('price');
+      }
+    }
```

Comment:
you don't have a variable called price. You would have to make it totalPrice or change your totalPrice variable to price